crates/apollo-compiler/tests/introspection.rs

```diff
@@ -0,0 +1,136 @@
+use apollo_compiler::execution::coerce_variable_values;
+use apollo_compiler::execution::JsonMap;
+use apollo_compiler::execution::Response;
+use apollo_compiler::execution::SchemaIntrospectionQuery;
+use apollo_compiler::ExecutableDocument;
+use apollo_compiler::Schema;
+use expect_test::expect;
+use expect_test::expect_file;
+
+#[test]
```

Comment:
sometime later we should port over the graphql-js introspection & execution tests